templates/functionaries.html

```diff
@@ -0,0 +1,126 @@
+{#-#################################################################
+<File Name>
+  functionaries.html
+
+<Author>
+  Lukas Puehringer <lukas.puehringer@nyu.edu>
+
+<Started>
+  Jun 30, 2017
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  Used to add new functionaries (by name) and upload functionary public key
+  using a file upload dropzone for each newly added functionary.
+
+#################################################################-#}
+{% import '_macros.html' as macros %}
+{% extends "base.html" %}
+
+{#- BEGIN: Functionary form macro -#}
+{% macro item_form(name=None, template=False) %}
+<div class="functionary dz-container p-3 mt-2 {{'template' if template}}">
+  <div class="m-2">
+    <span class="functionary-name">{{name}}</span><br>
+    <small>(Drop public key or click in box to upload)</small>
```

Comment:
I attempted to upload a public key, and it might have been an supported format because I got a popup error.  I don't know what the error was because the window disappeared after 1-3 seconds.

Reply:
Fixed with #18 and #21